activity_browser/ui/wizards/settings_wizard.py

```diff
@@ -109,22 +114,36 @@ def __init__(self, parent=None):
         self.bwdir.currentTextChanged.connect(self.bwdir_change)
         self.restore_defaults_button.clicked.connect(self.restore_defaults)
 
-    def bw_projects(self, path: str):
-        """Finds the bw_projects from the brightway2 environment provided by path"""
+    def bw_projects(self, path: str) -> List[str]:
```

Comment:
Can you help me understand this? The RE does... not much, and why wouldn't we just read the projects from the [`projects.db`](https://github.com/brightway-lca/brightway2-data/blob/wasm/bw2data/project.py#L70C35-L70C48) database. There isn't any guarantee that each directory is a project.

Reply:
So it looks like this `bw_projects` call was added after I stopped working on the AB and I can't explain why it was added here...

Will set up a separate 'unchanged' environment to see how it is being used.